deserialization/deserialize.py

```diff
@@ -19,26 +19,25 @@ class ResultType(Enum):
         formatter_class=argparse.ArgumentDefaultsHelpFormatter)
 
 parser.add_argument("--type", "-T", type=ResultType, dest="type",
-                    help=f"Type of results data to deserialize", required=True)
+                    help="Type of results data to deserialize", required=True)
 parser.add_argument("--input_file", "-f", type=Path, dest="input_file",
-                    help=f"Path to the data to deserialize", required=True)
+                    help="Path to the data to deserialize", required=True)
 
 def extractData(json_file: Path):
     # Parse the JSON document
-    with open(json_file, 'r', encoding='utf-8') as f:
-        buf = json.load(f)
+    root = json.load(json_file.load_text())
```

Comment:
`json.load()` accepts a file object, `json.loads()` is what loads from a string

```suggestion
    root = json.loads(json_file.load_text())
```

Reply:
ah. Of course. `s` for `string`, not for the verb.